src/index.html

```diff
@@ -7,10 +7,29 @@
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
+    <title>Search bar airbnb</title>
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="form-search form-search-big" data-qa="big">
+      <label class="form-search__label form-search__label-big">
+        <input
+        class="form-search__input form-search__input-big "
+         data-qa="keypress"
+         type="text"
+         placeholder="Try &quot;Los Angeles&quot;"
+         >
```

Comment:
You need to check and fix it everywhere
<img width="1049" alt="image" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/64750958/78eb8455-e78e-4a46-a582-8a03fd4e855e">